src/curve/stable/StablePair.sol

```diff
@@ -268,8 +268,8 @@ contract StablePair is ReservoirPair {
 
         if (timeElapsed > 0 && _reserve0 != 0 && _reserve1 != 0) {
             _updateOracle(
-                uint256(_reserve0) * token0PrecisionMultiplier,
-                uint256(_reserve1) * token1PrecisionMultiplier,
+                uint256(_reserve0),
+                uint256(_reserve1),
                 timeElapsed,
```

Comment:
What was the reason for the change, to align across pool types?

Reply:
yes it's to standardize across pool types. See the doc changes for `_updateOracle`.